src/java.management/share/classes/javax/management/openmbean/SimpleType.java

```diff
@@ -55,6 +55,9 @@
  * In other words, its methods <code>getClassName</code>, <code>getTypeName</code> and <code>getDescription</code>
  * all return the same string value <i>className</i>.
  *
+ * @param <T> the Java type that instances described by this type must
+ * have.
+ *
```

Comment:
I would suggest to say "that values described by this type"...

Reply:
Will change to "the Java type that values described by this SimpleType must have."